package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lumen5/framefusion",
-  "version": "0.0.12",
+  "version": "0.0.14",
```

Comment:
why is this jumping from 12 to 14?

Reply:
Because I was about to release 13 in another PR, so I bumped this 2 steps ahead to be ready 😄

Reply:
The other PR: https://github.com/Lumen5/framefusion/pull/29